ports/unix/README.md

```diff
@@ -78,7 +145,7 @@ Debug Symbols
 
 By default, builds are stripped of symbols and debug information to save size.
 
-To build a debuggable version of the Unix port, there are two options
+To build a debuggable version of the Unix port, there are two options:
 
 1. Run `make [other arguments] DEBUG=1`. Note setting `DEBUG` also reduces the
    optimisation level, so it's not a good option for builds that also want the
```

Comment:
While we are fixing this up, maybe worth mentioning that the optimization level can be controlled with `COPT`? In particular, I find it useful to use `COPT=-O0` so that nothing gets optimized out.

Reply:
@dlech Great idea, how does that look?

Reply:
`make [other arguments] DEBUG=1 COPT=-O0`

Reply:
@dlech oh sorry, I wasn't clear - I meant "I have pushed a commit with the extra docs, how does that look now?"

Reply:
Oops. 😄 

The extra section looks good to me.